app/src/main/java/com/example/androidinternshippart3/MainActivity.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.androidinternshippart3
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentTransaction
+import androidx.recyclerview.widget.RecyclerView
+import com.example.androidinternshippart3.login.LoginFragment
+import com.example.androidinternshippart3.register.RegisterFragment
+import com.google.android.material.tabs.TabLayout
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        supportActionBar?.hide()
+        setContentView(R.layout.activity_main)
+        var count = 0
+
+        tabLayout.addOnTabSelectedListener(object : TabLayout.OnTabSelectedListener {
+
+            override fun onTabSelected(tab: TabLayout.Tab?) {
+            Log.d("count",count++.toString())
```

Comment:
There is two tab in design, var count is not needed. Check which fragment instance current now and switch to another.